lib/TelemetryLog.js

```diff
@@ -2,9 +2,21 @@ const simpleStorage = require('sdk/simple-storage');
 const xutils = require('shield-studies-addon-utils');
 
 class TelemetryLog {
+  get logData() {
+    return simpleStorage.storage.telemetryLogData;
+  }
+
+  set logData(newObject) {
+    simpleStorage.storage.telemetryLogData = newObject;
+  }
+
+  increaseByOne(prop) {
```

Comment:
This is unnecessary, you should just be able to use `+=` directly on the object returned by `logData`.